Dockerfile

```diff
@@ -195,25 +212,50 @@ RUN chown deploy:deploy $(pwd)
 
 USER deploy
 
+# Production node modules.
+COPY --chown=deploy:deploy --from=yarn-deps /opt/node_modules.prod ./node_modules
+
 # Base code
 COPY --chown=deploy:deploy --from=builder /app/data ./data
 COPY --chown=deploy:deploy --from=builder /app/package.json .
 COPY --chown=deploy:deploy --from=builder /app/scripts ./scripts
 COPY --chown=deploy:deploy --from=builder /app/webpack ./webpack
 COPY --chown=deploy:deploy --from=builder /app/yarn.lock .
 
+# ---------------------------------------------------------
+# Release (Novo) image
+# ---------------------------------------------------------
+FROM production-base as production-novo
+
 # Client assets
 COPY --chown=deploy:deploy --from=builder /app/manifest.json .
 COPY --chown=deploy:deploy --from=builder /app/public ./public
 COPY --chown=deploy:deploy --from=builder /app/src ./src
 
 # Server assets
-COPY --chown=deploy:deploy --from=builder /app/server.dist.js .
-COPY --chown=deploy:deploy --from=builder /app/server.dist.js.map .
+COPY --chown=deploy:deploy --from=builder /app/server-novo.dist.js .
+COPY --chown=deploy:deploy --from=builder /app/server-novo.dist.js.map .
 
+ENTRYPOINT ["/usr/bin/dumb-init", "--"]
+CMD ["node", "--max_old_space_size=3072", "-r", "dotenv/config", "./server-novo.dist.js"]
 
-# Production node modules.
-COPY --chown=deploy:deploy --from=yarn-deps /opt/node_modules.prod ./node_modules
+# ---------------------------------------------------------
+# Release image
+# ---------------------------------------------------------
+#
+# Release stage. This stage creates the final docker iamge that will be
+# released. It contains only production dependencies and artifacts.
+#
+FROM production-base as production
+
+# Client assets
+COPY --chown=deploy:deploy --from=builder /app/manifest.json .
+COPY --chown=deploy:deploy --from=builder /app/public ./public
+COPY --chown=deploy:deploy --from=builder /app/src ./src
+
+# Server assets
+COPY --chown=deploy:deploy --from=builder /app/server.dist.js .
+COPY --chown=deploy:deploy --from=builder /app/server.dist.js.map .
 
 ENTRYPOINT ["/usr/bin/dumb-init", "--"]
-CMD ["yarn", "start"]
+CMD ["node", "--max_old_space_size=3072", "-r", "dotenv/config", "./server.dist.js"]
```

Comment:
@eessex - The changes we've made to `staging.yml/production.yml` create a 2nd set of pods, for novo. The pods will run the same Docker image (as the canonical pods). Shall they run the same `CMD`? Or, should we over-write it (can be done in k8s yml)? Or, were we planning to use the same `CMD` and toggle app-selection by ENV var, in that case, we can set the var (also can be done in k8s yml).